static/js/app.js

```diff
@@ -178,30 +69,30 @@ async function addLifeEvents() {
       ${event.more ? `<p>More: ${event.more}</p>` : ''}
     </div>
   `).join('');
-  console.log("life events",lifeEventsHtml)
+  console.log(`${eventType} events`, eventsHtml)
   // Append the generated HTML to the container
-  lifeEventsContainer.innerHTML = lifeEventsHtml;
+  eventsContainer.innerHTML = eventsHtml;
 }
 
 ////////////////////////Execute methods\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
-//DOM Listener makes sure all the content is loaded befere execute the methods
+//DOM Listener makes sure all the content is loaded before execute the methods
 document.addEventListener('DOMContentLoaded', async () => {
```

Comment:
Just an idea 💡. Have you thought of defining your hierarchy in as a JSON and using it to call the appropriate `loadEventHtml` or `addEvents` function? 

Something like: 

```json
[
  {
    "eventTypeName": "aboutMe", 
    "types": []
  },
  {
    "eventTypeName": "experience", 
    "type": ["Work"]
  }, 
  {
    "eventTypeName": "education", 
    "type": ["Learn"]
  }, 
  {
    "eventTypeName": "projects", 
    "type": ["Project"]
  }, 
  {
    "eventTypeName": "life", 
    "type": ["Life"]
  }
]
```

Reply:
This would be the code of the whole function instead:

```js
var hierarchy = [ {...} ]; // as defined in the comment above
for (var et in hierarchy) {
  loadEventHtml(et.eventTypeName)
  for (var t in et.types) {
    addEvents(et.eventTypeName, t);
  }
}
```

Reply:
Great idea! Already implemented it!